File: 1st/Jaehee/원소들의 곱과 합.java
class Solution {
    public int solution(int[] num_list) {
        int product = 1; 
        int sum = 0;     
        
        for (int num : num_list) {
            product *= num;
            sum += num;
        }
        
        return product < (sum * sum) ? 1 : 0;
    }
}

File: 3st/Juram/programmer/원소들의 곱과 합.js
function solution(num_list) {
    let a = 0;
    let b = 1;
    for(let num of num_list){
        a+=num;
        b*=num;
    }
    if(b<a*a){
        return 1;
    }
    else return 0;
}